deployment/constructs/lambda-dynamo-event.ts

```diff
@@ -0,0 +1,102 @@
+import { Construct } from "constructs";
+import {
+  aws_dynamodb as dynamodb,
+  aws_iam as iam,
+  aws_lambda as lambda,
+  aws_lambda_nodejs as lambda_nodejs,
+  Duration,
+} from "aws-cdk-lib";
+
+interface LambdaDynamoEventProps
+  extends Partial<lambda_nodejs.NodejsFunctionProps> {
+  additionalPolicies?: iam.PolicyStatement[];
+  brokerString?: string;
+  handler: string;
+  iamPath: string;
+  iamPermissionsBoundary: iam.IManagedPolicy;
+  memorySize?: number;
+  stackName: string;
+  tables: dynamodb.Table[];
+  timeout?: Duration;
```

Comment:
```suggestion
  iamPath: string;
  iamPermissionsBoundary: iam.IManagedPolicy;
  stackName: string;
  tables: dynamodb.Table[];
```
Looks like these ones are already on [NodejsFunctionProps](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda_nodejs.NodejsFunctionProps.html)